src/main/java/com/github/mizool/rest/errorhandling/AbstractErrorHandlingFilter.java

```diff
@@ -0,0 +1,58 @@
+package com.github.mizool.rest.errorhandling;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+import com.github.mizool.rest.core.TransactionalResponseWrapper;
+
+public abstract class AbstractErrorHandlingFilter implements Filter
+{
+    protected abstract byte[] getErrorMessageJsonBytes(ErrorMessageDto errorMessageDto);
```

Comment:
methods go below fields. put the method below `doFilter()`as that method is "entry point" for readers, not this method.